src/sales/order-workflow.md

```diff
@@ -8,13 +8,13 @@ Step 1: Place Order
 : The checkout process begins when the shopper clicks the Go to Checkout button on the shopping cart page or [reorders]({% link sales/reorders-allow.md %}) directly from their customer account.
 
 Step 2: Order Pending
-: In the Orders grid, the status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled.
+: The status of the sales order is initially `Pending`. Payment has not been processed, and the order can still be edited or canceled. This will occur when payment processor module is configured for authorization mode.
 
 Step 3: Receive Payment
-: The status of the order changes to `Processing` when payment is received or authorized. Depending on the payment method, you might receive notification when the transaction is authorized or processed.
+: The status of the order changes to `Processing` when payment is received or authorized. Depending on the payment method, you might receive notification when the transaction is authorized or processed. This will occur automatically when payment processor module is configured for capture or intent sale mode.
```

Comment:
```suggestion
: The order status changes to `Processing` when payment is received or authorized. Depending on the payment method, you might receive notification when the transaction is authorized or processed. This will occur automatically when the payment method is configured for capture or intent sale mode.
```

Reply:
What is intent sale mode? I may just not be familiar with this term.